docs/content/preview/yugabyte-platform/anywhere-automation/anywhere-cli.md

```diff
@@ -0,0 +1,273 @@
+---
+title: yba CLI
+headerTitle: YugabyteDB Anywhere CLI
+linkTitle: yba CLI
+description: Use YugabyteDB Anywhere CLI to create and manage universes.
+headcontent: Install yba, configure default settings, and set up autocompletion
+earlyAccess: /preview/releases/versioning/#feature-availability
+menu:
+  preview_yugabyte-platform:
+    parent: anywhere-automation
+    identifier: anywhere-cli
+    weight: 50
+type: docs
+rightNav:
+  hideH4: true
+---
+
+The [YugabyteDB Anywhere (YBA) Command Line Interface](https://github.com/yugabyte/yugabyte-db/tree/master/managed/yba-cli) (yba) is an open source tool that enables you to interact with YBA accounts using commands in your command-line shell. With minimal configuration, you can start running commands from the command line that implement functionality equivalent to that provided by the browser-based YBA interface.
```

Comment:
The [YugabyteDB Anywhere (YBA) Command Line Interface (CLI)] (https://github.com/yugabyte/yugabyte-db/blob/master/managed/yba-cli/docs/yba.md) is an open source tool that enables you to interact with YBA using commands from your shell. With...

_Updated link and some text here. Can you verify if this is ok_

Reply:
One more line to add.

_Note: The CLI can interact only with a version of YBA 2024.1 or greater._

Reply:
Wondering whether we should also add a one-liner that says _"Note: This is an early access version of the CLI. The commands and options could change in a later stable version of the CLI"._

Reply:
Sure. And sounds like a tech preview instead of early access